android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -258,3 +255,39 @@ public void enable(final Boolean enabled) {
     return constants;
   }
 }
+
+class OnCompletionListenerLooped implements OnCompletionListener {
```

Comment:
Why static? Nobody gonna use it outside the RNSoundModule and static is little heavier than private. Isn't it? I'm not java master.